components/scenario-screen/display-step-credentials.tsx

```diff
@@ -3,22 +3,24 @@ import { Trash2, Edit } from "lucide-react";
 import { Button } from "@/components/ui/button";
 import { NoSelection } from "../credentials/no-selection";
 import { EditProofRequest } from "./edit-proof-request";
-import { ProofRequestAttributes, ProofRequestPredicates, ShowcaseJSON } from "@/types";
+import {
+  ProofRequest,
+  ProofRequestAttributes,
+  ProofRequestPredicates,
+  ShowcaseJSON,
+} from "@/types";
 import { cn } from "@/lib/utils";
 
 interface DisplayStepCredentialsProps {
   selectedCharacter: number;
   showcaseJSON: ShowcaseJSON;
   localData: {
     requestOptions?: {
-      proofRequest?: {
-        attributes?: { [key: string]: ProofRequestAttributes };
-        predicates?: { [key: string]: ProofRequestPredicates };
-      };
+      proofRequest?: ProofRequest;
     };
   };
-  selectedStep: number;
-  selectedScenario: number;
+  selectedStep: number | null;
```

Comment:
just checking, any specific reason why we are using null here?

Reply:
We can have no selected steps in order to display`<NoSelection`